lib/fizzy/capi.cpp

```diff
@@ -162,9 +162,8 @@ inline std::vector<fizzy::ExternalFunction> unwrap(
 inline fizzy::ImportedFunction unwrap(const FizzyImportedFunction& c_imported_func)
 {
     fizzy::ImportedFunction imported_func;
-    imported_func.module =
-        c_imported_func.module ? std::string{c_imported_func.module} : std::string{};
-    imported_func.name = c_imported_func.name ? std::string{c_imported_func.name} : std::string{};
+    imported_func.module = c_imported_func.module;
```

Comment:
https://stackoverflow.com/a/10771938

But we require `module` and `name` to be non-null, right?

Reply:
Right, that's why it was unnecessary check.